tests/rbd/rbd_utils.py

```diff
@@ -104,6 +104,8 @@ def check_pool_exists(self, pool_name: str) -> bool:
         Returns:  True -> pass, False -> fail
 
         """
+        # sleeping for 5 seconds for the status of pool to be updated in ceph df command
+        sleep(5)
```

Comment:
Can we loop to check if the pool status is updated instead of a blind sleep?

Reply:
made changes to check status of pool every 2 seconds. Total wait time for pool creation : 200 seconds.